addons/website_forum/tests/test_performance.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website.tests.test_performance import UtilPerf
+
+import random
+
+
+class TestForumPerformance(UtilPerf):
+    def setUp(self):
+        super().setUp()
+        # if website_livechat is installed, disable it
+        if 'channel_id' in self.env['website']:
+            self.env['website'].search([]).channel_id = False
+
+        self.base_website = self.env.ref("website.default_website")
+        self.base_forum = self.env.ref("website_forum.forum_help")
+
+        Forum = self.env['forum.forum']
+        Post = self.env['forum.post']
+
+        TestUsersEnv = self.env['res.users'].with_context({'no_reset_password': True})
+        group_employee_id = self.env.ref('base.group_user').id
+        self.user_employee = TestUsersEnv.create({
+            'name': 'Armande Employee',
+            'login': 'Armande',
+            'email': 'armande.employee@example.com',
+            'karma': 0,
+            'groups_id': [(6, 0, [group_employee_id])]
+        })
+
+        # remove menu containing a slug url (only website_helpdesk normally), to
+        # avoid the menu cache being disabled, which would increase sql queries
+        self.env['website.menu'].search([
+            ('url', '=like', '/%/%-%'),
+        ]).unlink()
+
+        self.forum = Forum.create({
+            'name': 'TestForum',
+        })
+        self.post = Post.create({
+            'name': 'TestQuestion',
+            'content': 'Who\'s afraid of Virginia Wolf?',
```

Comment:
```suggestion
            'content': 'Who's afraid of Virginia Wolf?',
```
note that some comments might apply to multiple places in the PR